cfme/networks/views.py

```diff
@@ -555,7 +555,7 @@ class SubnetDetailsToolBar(View):
 class SubnetAddView(BaseLoggedInPage):
     """ Represents Add view of subnet """
     title = Text('//div[@id="main-content"]//h1')
-    network_manager = Select(id='ems_id')
+    network_manager = Select(name='ems_id')
```

Comment:
Question : Is this both for 5.9 and 5.10 ?

Reply:
Yes the same for 5.8, 5.9 and 5.10